next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsSection.tsx

```diff
@@ -1,18 +1,70 @@
 import MyApplicationsHeader from 'components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsHeader'
-import { useGlobalStateContext } from 'components/forms/states/GlobalState'
+import { useRouter } from 'next/router'
 import { useTranslation } from 'next-i18next'
+import { useEffect, useState } from 'react'
+import { useEffectOnceWhen } from 'rooks'
 
 import MyApplicationsList from './MyApplicationsList'
 
+const sections = ['SENT', 'SENDING', 'DRAFT'] as const
```

Comment:
would make this lowercase and in slovak, since that's how the rest of the url pahts looks like (for the better or worse):

```suggestion
const sections = ['odoslane', 'odosiela-sa', 'koncepty'] as const
```